packages/react/src/components/Tabs/TabList.tsx

```diff
@@ -27,6 +27,7 @@ export const TabList = forwardRef<
       role='tablist'
       activeValue={value}
       className={cl('ds-tabs__tablist', className)}
+      orientation='ambiguous'
```

Comment:
I think you in this case should just remove `orientation`? ☺️

Reply:
The default orientation on `RovingFocusRoot` is `horizontal`, so we need to explicitly set `ambiguous` here